packages/vscode-dependencies-validation/test/moduleProxies.ts

```diff
@@ -12,5 +12,38 @@ export const npmDepsValidationProxy = {
   findDependencyIssues() {
     return Promise.reject("findDependencyIssues method is not implemented");
   },
+  isPathExist() {
+    return Promise.reject("isPathExist method is not implemented");
+  },
+  "@noCallThru": true,
+};
+
+export const eventUtilProxy = {
+  handlePackageJsonEvent() {
+    return Promise.reject("handlePackageJsonEvent method is not implemented");
+  },
+  "@noCallThru": true,
+};
+
+export const configurationProxy = {
+  isAutoFixEnabled() {
+    throw new Error("configurationProxy method is not implemented");
+  },
+  "@noCallThru": true,
+};
+
+export const utilProxy = {
```

Comment:
so it seems strange that we need to proxy our own module, particularly when it
is not using un-aviliable imports (`vscode`). So I looked at one of the tests that uses 
this proxy and it does not look like a "Black Box" unit test, so maybe this proxy won't be needed.

I will give more details on a specific test in another comment